src/fontstash.h

```diff
@@ -781,6 +822,12 @@ void fonsSetFont(FONScontext* stash, int font)
 	fons__getState(stash)->font = font;
 }
 
+void fonsSetOversample(FONScontext* stash, int oversampleH, int oversampleV)
+{
+	fons__getState(stash)->oversampleH = oversampleH;
```

Comment:
Re: validation, I tend to add some when I think that there's a good sane range and when values outside that range can have big negative impact elsewhere (i.e. oversample = 1000).